src/components/Payoff/index.en.js

```diff
@@ -1,20 +1,20 @@
 import React from 'react'
 import { StyledContainer, StyledSubParagraph, StyledTextContainer, StyledHeading, StyledParagraph, StyledIntro, StyledImage } from './index.styles.js'
 import ResponsiveImage from '../ResponsiveImage'
-import missedChances from '../../images/missed-chances.svg'
+import missedChances from '../../images/city.png'
 
 const Payoff = () => (
   <StyledContainer>
     <StyledIntro>
       <StyledTextContainer>
-        <StyledHeading>Signalen</StyledHeading>
-        <StyledParagraph>Working together for quality of life</StyledParagraph>
-        <StyledSubParagraph>Signalen is an open source process and task system for governments, which automatically categorizes and routes reports about public space for processing by the appropriate handler.</StyledSubParagraph>
-        <StyledSubParagraph><a href="/en/#about-signalen">More information on Signalen</a></StyledSubParagraph>
+        <StyledHeading>Save the date: Signalen Community Meetup</StyledHeading>
+        <StyledParagraph>Online edition. March, 22. </StyledParagraph>
+        <StyledSubParagraph>This edition is specifically for municipalities that are considering using Signalen to receive and handle their nuisance reports. In addition, during the meetup we will explore with the market how to ensure a sustainable ecosystem around Signalen.</StyledSubParagraph>
+        <StyledSubParagraph><a href="/en/news/2021-02-26-save-the-date-community-meeting">More information on the meetup</a></StyledSubParagraph>
       </StyledTextContainer>
     </StyledIntro>
     <StyledImage>
-      <ResponsiveImage src={missedChances} alt="Illustration of man with headphone and woman with mobile phone walking down the street" />
+      <ResponsiveImage src={city} alt="City with construction workers, people walking by and a hadicapped man" />
```

Comment:
@AlbaRoza great PR! There is one small problem. You are referencing a variable {city} which does not exist and will break the build. Consider changing either line 4 with `import missedChances` to `import city` OR chance in this line {city} to {missedChances}. If you need help let me know.

Reply:
@AlbaRoza I made the changes for you.